data/colors/vars/component_light.ts

```diff
@@ -187,5 +187,9 @@ export default {
       bg: get('scale.blue.5'),
       disabledBg: get('scale.gray.5'),
     }
+  },
+
+  control: {
+    border: get('scale.gray.4') // maintains min 3:1 contrast ratio
```

Comment:
I've only tested this in `light` but we should be able to slightly lighten this and still meet the minimum requirement: ``lighten(get('scale.gray.4'), -0.03)``